gb-cpu/src/microcode/controller.rs

```diff
@@ -104,13 +104,14 @@ impl MicrocodeController {
                 fetch
             }
         });
+        let mut state = State::new(regs, bus, int_flags.clone());
 
```

Comment:
`state` is needed on line 102 in the feature `registers_logs`

Reply:
it's a clojure, the clojure require the state but only during it's execution

Reply:
Ok but how can line 102 access something initialised line 107, I don't get it

Reply:
done